mycroft/configuration/config.py

```diff
@@ -128,17 +132,20 @@ class ConfigurationManager(object):
     _config = None
 
     @staticmethod
-    def load(*config_files):
+    def load(defaults=CONFIG_LOCATIONS, *config_files):
```

Comment:
looking back now, I'm surprised that python allows this. My understanding is that you can't have any non-keyword arguments (and varargs definitely fit the bill for that) after kwargs (defaults= being a kwarg).